src/components/admin/Admin.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import UsersTable from './UsersTable';
+import ActionTable from './ActionTable';
+import PendingTable from './PendingTable';
+import AdminTopBar from './AdminTopBar';
+import styles from './admin.module.css';
+
+
+//class that handles the front page of the admin interface
```

Comment:
```suggestion
// Class that handles the front page of the admin interface
```
Or you can remove this comment. In my opinion, the name of class is clear :)